app/views/liquids/_github_code.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="ltag-github-code-tag">
```

Comment:
I wonder if the UI could have two improvements:

1. a header similar to this:

![Screenshot_2020-06-01 New Post - DEV(local) Community 👩‍💻👨‍💻](https://user-images.githubusercontent.com/146201/83399558-b5c91c00-a401-11ea-8619-15c624b442bd.png)

(you can get the style from `GithubReadmeTag.scss`)

2. (totally optional and can be done in another PR): syntax highlighting. 

Unfortunately GitHub's contents API doesn't return the detected language it seems: https://developer.github.com/v3/repos/contents/#get-contents

Reply:
> 1. a header similar to this:

When you're saying a header similar to the GitHubReadme you're referring the path having different links for each part? The way I wrote the second line was based on Github's way of rendering code snippets.

> 2. syntax highlighting

If I remember correctly, passing the file type through the `HTMLRogue` will provide the syntax highlighting, the screenshot was using `.vscodeignore` that might not have highlight support. I  will need to fix my current dev environment (WSL + Windows), to get the page preview working again to confirm the behavior.

Reply:
Hey @rhymes, may I know what you're referring to when you said a header similar to the screenshot above?